ui/src/GroupAdmin.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useRef, useState } from "react";
+import { useHistory } from "react-router-dom";
+import { Helmet } from "react-helmet";
+import Announcer from "react-a11y-announcer";
+import { Link } from "react-router-dom";
+
+function GroupAdmin(props) {
+  let history = useHistory();
+  const [userTableRows, setUserTableRowsRows] = useState([]);
+
+  function openRemoveUserPage(groupUser) {
+    history.push(
+      `deleteuserfromgroupconfirmation?groupUserId=${groupUser.id}&groupName=${props.groupName}&groupId=${props.groupId}&userEmail=${groupUser.userEmail}`
+    );
+  }
+
+  useEffect(() => {
+    async function fetchAllUsersInGroup() {
+      const allGroupMembersResponse = await fetch(
+        `/api/userGroupMembers/findByGroup/${props.groupId}`
+      );
+      const allUsersInGroup = await allGroupMembersResponse.json();
+
+      const usersInGroupRows = await allUsersInGroup.map((groupUser, index) => (
+        <tr className="table__row" key={index}>
+          <td className="table__cell">{groupUser.userEmail}</td>
+          <td className="table__cell">
+            <button onClick={() => openRemoveUserPage(groupUser)} type="button">
```

Comment:
this button does not follow any government standards (GDS) or the ONS pattern library... it's just a completely unstyled plain HTML button.